Projects/Checkers/Board.cs

```diff
@@ -0,0 +1,171 @@
+﻿namespace Checkers;
+
+public class Board
+{
+	public List<Piece> Pieces { get; set; }
+
+	public Piece? Aggressor { get; set; }
+
+	public Piece? this[int x, int y] =>
+		Pieces.FirstOrDefault(piece => piece.X == x && piece.Y == y);
+
+	public Board()
+	{
+		Aggressor = null;
+		Pieces = new List<Piece>
+			{
+				new() { NotationPosition ="A3", Color = Black},
+				new() { NotationPosition ="A1", Color = Black},
+				new() { NotationPosition ="B2", Color = Black},
+				new() { NotationPosition ="C3", Color = Black},
+				new() { NotationPosition ="C1", Color = Black},
+				new() { NotationPosition ="D2", Color = Black},
+				new() { NotationPosition ="E3", Color = Black},
+				new() { NotationPosition ="E1", Color = Black},
+				new() { NotationPosition ="F2", Color = Black},
+				new() { NotationPosition ="G3", Color = Black},
+				new() { NotationPosition ="G1", Color = Black},
+				new() { NotationPosition ="H2", Color = Black},
+
+				new() { NotationPosition ="A7", Color = White},
+				new() { NotationPosition ="B8", Color = White},
+				new() { NotationPosition ="B6", Color = White},
+				new() { NotationPosition ="C7", Color = White},
+				new() { NotationPosition ="D8", Color = White},
+				new() { NotationPosition ="D6", Color = White},
+				new() { NotationPosition ="E7", Color = White},
+				new() { NotationPosition ="F8", Color = White},
+				new() { NotationPosition ="F6", Color = White},
+				new() { NotationPosition ="G7", Color = White},
+				new() { NotationPosition ="H8", Color = White},
+				new() { NotationPosition ="H6", Color = White}
+			};
+	}
+
+	public static string ToPositionNotationString(int x, int y)
+	{
+		if (!IsValidPosition(x, y)) throw new ArgumentException("Not a valid position!");
+		return $"{(char)('A' + x)}{y + 1}";
+	}
+
+	public static (int X, int Y) ParsePositionNotation(string notation)
+	{
+		if (notation is null) throw new ArgumentNullException(nameof(notation));
```

Comment:
you _could_ explicitly throw this exception but it is redundant as it will get thrown anyway if `notation is null`

Reply:
A `ArgumentNullException` is a little nicer than a `NullReferenceException`, so I would prefer to keep in, even though based on the current application it will never have invalid input (since that method is only called with valid hard-coded input).